muxinc/elements
# feat/mux-video-react: initial impl of mux-video-react. Basic CRA yarn…

…+ts example package. Basic next.js yarn+ts example package.

Creating a draft PR for starting review and conversation at a high level.

To test:
* for `mux-video-react`, install deps via `yarn` and then run either `yarn dev` (for watch mode rebuilds) or `yarn build`
* in the new `/examples` directory, you should be able to install and run either/both `create-react-app-with-typescript` or `nextjs-with-typescript`, which has a basic demo of using `MuxVideo`.
